bigtable/autoscaler/autoscaler.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample that demonstrates how to use Bigtable Stackdriver metrics to
+autoscale Google Cloud Bigtable."""
+
+import argparse
+import os
+import time
+
+from google.cloud import bigtable
+from google.cloud import monitoring
+
+import strategies
+
+CPU_METRIC = 'bigtable.googleapis.com/cluster/cpu_load'
```

Comment:
while constants are good practice in production code, in sample code they add a layer of indirection that can be annoying when including a snippet in cloud site. Strongly prefer this to go near where it's used.